src/App.jsx

```diff
@@ -14,33 +16,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const reset = () => {
+    setSortField('');
+    setReversed(false);
+  };
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  // Створення копії масиву перед сортуванням
+  const sortedGoods = [...goodsFromServer].sort((good1, good2) => {
+    switch (sortField) {
+      case 'length':
+        return good1.length - good2.length;
+      case 'alphabet':
+        return good1.localeCompare(good2);
+      default:
+        return 0;
+    }
+  });
```

Comment:
The sorting logic is correctly implemented, but it might be beneficial to handle the case where `sortField` is neither 'length' nor 'alphabet' more explicitly. Currently, it defaults to returning 0, which means no sorting is applied. Consider adding a comment or handling this case more explicitly if needed.